docs/plugins/npe2_manifest_specification.md

```diff
@@ -0,0 +1,606 @@
+(npe2-manifest-spec)=
+
+# npe2 manifest specification
+
+```{admonition} Backward compatibility
+:class: tip
+We introduced a new plugin engine in December 2021. The new library [`npe2`][npe2]
+is a re-imagining of how napari interacts with plugins. Plugins targeting
+`napari-plugin-engine` will continue to work, but we recommend migrating to
+`npe2` as soon as possible.  See the [](npe2-migration-guide).
+```
+
+The **plugin manifest** is a specially formatted text file declaring the
```

Comment:
we've already said this elsewhere (in the entry page).  I'd love to see our documentation be _leaner_, which means we don't really need someone to be able to enter anywhere and bring them back up to speed.  Let the top pages give the background, the inner pages be the reference, and the tutorial pages read like hand-holding how-to

Reply:
I think I might have removed this from the plugin index page. I have it here because this is where I define all these terms.